src/AspNetCoreModuleV2/RequestHandlerLib/application.h

```diff
@@ -17,9 +18,18 @@ class APPLICATION : public IAPPLICATION
         return m_status;
     }
 
-    APPLICATION()
+    APPLICATION(REQUESTHANDLER_CONFIG* pConfig)
         : m_cRefs(1)
     {
+        m_pConfig = pConfig;
```

Comment:
Please don't do manual memory management. You can also just pass the application path in.